cli/hoodie-defaults.js

```diff
@@ -9,6 +9,7 @@ function getHoodieDefaults () {
     dbUrl: undefined,
     inMemory: false,
     loglevel: 'warn',
-    url: undefined
+    url: undefined,
+    applicationRoot: undefined
```

Comment:
> if an application module that defined hoodie's behaviour was being included as a dependency of another module

If we enforce the folder structure so that client/server plugins can be required with `require(module-name-here/hoodie/client)` and `require(module-name-here/hoodie/server)` then we should be good. If the an app is included as a dependency, that the plugins should also be loaded as third party plugins, right?

> if an application defined it's own cli application available via the PATH

I can’t follow you hear. What do you mean with "if an application defined its own CLI application"?

When in doubt, I would rather keep options to a minimum and only add them if we there are specific use cases. Because removing options is much harder than adding them later on